app/api/v2/views/orders.py

```diff
@@ -0,0 +1,101 @@
+from flask_restplus import Resource, Namespace, reqparse, fields
+from flask import request
+from app.api.utils.parcel_validator import ParcelSchema, DestinationParser
+from ..models.orders_model import OrderModel
+from ..models.user_model import UserModel
+from marshmallow import post_load
+#
+# db = OrderModel()
+# user_db = UserModel()
+v2_order = Namespace('parcels')
+new_order = v2_order.model('Orders', {
+    'sender_name': fields.String(description="John Doe"),
+    'receiver_name': fields.String("Alfie kavaluku"),
+    'receiver_contact': fields.String("alfie@gmail.com"),
+    'weight': fields.Integer(10),
+    'pickup_location': fields.String('kiambu'),
+    'destination': fields.String("nairobi")
+})
+update_destination = v2_order.model('order', {
+    'destination': fields.String(description='destination')
+})
+
+
+class Order(Resource):
+    """ class that owns the routes to create and get all parcels"""
+
+    def get(self):
+        """ route that returns all the parcels in the database"""
+        response = db.get_all_order()
+        return response, 200
+
+    @v2_order.expect(new_order)
+    @post_load
+    def post(self):
+        """route that allows users to create parcel delivery order"""
+        if not request.is_json:
```

Comment:
Your code is too much squeezed, therefore, you should always leave a line space before you start a new block of code like `for loop`, `if statements` and `return statements`. This will make your code more readable.